ui/src/message_popup/experience_page.jsx

```diff
@@ -46,42 +44,35 @@ export default React.createClass({
 
   getInitialState: function() {
     const isSolutionMode = _.has(this.props.query, 'solution');
-    const helpType = isSolutionMode ? 'none' : 'feedback';
-    const shouldShowStudentCard = isSolutionMode ? _.has(this.props.query, 'cards') : true;
-    const shouldShowSummary = !isSolutionMode;
-    const sessionId = uuid.v4();
     return {
-      shouldShowStudentCard,
-      shouldShowSummary,
-      helpType,
-      isSolutionMode,
-      sessionId,
-      competencyGroupValue: ALL_COMPETENCY_GROUPS,
-      questions: allQuestions,
-      email: this.context.auth.userProfile.email,
-      hasStarted: false,
-      questionsAnswered: 0,
-      sessionLength: 10,
+      scaffolding: {
+        email: this.context.auth.userProfile.email,
```

Comment:
Some of these seem like they're logically part of a session, rather than scaffolding (eg., `email`, `question` and `sessionLength`.  Does it make sense to move them there?